### PR TITLE
lag heroes on tb minimap using regular game flagging

### PR DESCRIPTION
**Pull request by DubbleClick**
the dirtiest commit in history

### CODE REVIEW

File: GWToolboxdll/Widgets/Minimap/Minimap.cpp [hunk 1]
```diff
@@ -110,6 +111,27 @@ void Minimap::SkillActivateCallback(GW::HookStatus*, GW::Packet::StoC::SkillActi
...
+
+/*
+ return std::tuple<is_flagging, hero_num>
+ todo: very dirty please fix papa jon
+ */
+std::pair<bool, int> Minimap::Flagging()
+{
+    const auto handle = reinterpret_cast<uintptr_t>(GetModuleHandle("Gw.exe"));
+    const auto* flagging_addr = reinterpret_cast<bool*>(handle + 0x637568);
+    const auto offsets = std::array{0x3C, 0x14, 0x24, 0x2C, 0x4};
```
Comment (reduf) [inline GWToolboxdll/Widgets/Minimap/Minimap.cpp:123]: No no no. This is not how we do stuff, take a look around.

Comment (DubbleClick) [reply GWToolboxdll/Widgets/Minimap/Minimap.cpp:123]: this isn't meant to be merged yet, it's just here so papa jon can take a look and possibly have an easier time

File: GWToolboxdll/Widgets/Minimap/Minimap.cpp [hunk 2]
```diff
@@ -110,6 +111,27 @@ void Minimap::SkillActivateCallback(GW::HookStatus*, GW::Packet::StoC::SkillActi
         }
     }
 }
+
+/*
+ return std::tuple<is_flagging, hero_num>
+ todo: very dirty please fix papa jon
```
Comment (DubbleClick) [inline GWToolboxdll/Widgets/Minimap/Minimap.cpp:117]: I thought this would give it away that this is not a serious PR yet

**Comment by 3vcloud** [comment]
This has been sorted in dev branch

**Pull request closed without merging**